docs/topics/i18n/translation.txt

```diff
@@ -1102,6 +1110,13 @@ directories listed in :setting:`LOCALE_PATHS` have the highest precedence with
 the ones appearing first having higher precedence than the ones appearing
 later.
 
+.. admonition:: i18n_patterns
+
+    If you are defining your URLs within :func:`~django.conf.urls.i18n.i18n_patterns`
+    in :setting:`ROOT_URLCONF`, you will also need to place the ``javascript_catalog``
```

Comment:
Would it be better to talk about the *root URLconf* instead of linking the `ROOT_URLCONF`setting here?.